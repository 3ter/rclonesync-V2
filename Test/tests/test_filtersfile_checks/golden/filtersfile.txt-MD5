3ccb786b07842d8208d971bf04340ee1