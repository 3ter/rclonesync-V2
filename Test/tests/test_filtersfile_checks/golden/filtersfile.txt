# Test filters file - DIFFERENT
#  Note that this test checks for rclonesync's access to a filters file, not an extensive test of rclone's filter capability

# Exclude fileZ.txt in root only.  The copy in the subdir should be found and synced.
- /fileZ.txt
